src/bin/cargo/commands/add.rs

```diff
@@ -100,7 +100,11 @@ Example uses:
                 .value_name("PATH")
                 .help("Filesystem path to local crate to add")
                 .group("selected")
-                .conflicts_with("git"),
+                .conflicts_with("git")
+                .add(clap_complete::engine::ArgValueCompleter::new(
+                    clap_complete::engine::PathCompleter::any()
+                        .filter(|path| path.join("Cargo.toml").exists()),
+                )),
```

Comment:
For future changes, should we refactor out these `--path` completers into https://github.com/rust-lang/cargo/blob/master/src/cargo/util/command_prelude.rs ?  We have two already and I believe there is a third *and* we'll be wanting to add more context to these